ui/components/MeshModelRegistry/MesheryTreeView.js

```diff
@@ -110,10 +110,10 @@ const MesheryTreeView = ({
 
   const handleScroll = (scrollingView) => () => {
     const div = event.target;
-    if (div.scrollTop >= div.scrollHeight - div.clientHeight - 2) {
+    if (div.scrollTop >= div.scrollHeight - div.clientHeight - 10) {
       setPage((prevPage) => ({
-        ...prevPage, // Keep the current values for other keys
-        [scrollingView]: prevPage[scrollingView] + 1, // Increment the specific key based on the view
+        ...prevPage,
```

Comment:
We lost a comment, @ptdatta. Please re-comment the code.

Reply:
Done.✅